src/fetcher/create-breadcrumbs.tsx

```diff
@@ -109,20 +120,19 @@ export function createBreadcrumbs(uuid: MainUuidType, instance: Instance) {
     instance: Instance
   ) {
     if (taxonomyPaths === undefined) return undefined
-    const breadcrumbCandidates = taxonomyPaths.map((candidate) =>
-      taxonomyParentsToRootToBreadcrumbsData(candidate, instance)
-    )
-    let breadcrumbs
+    let mainTax
+    let count = -1
 
     // select first shortest taxonomy path
```

Comment:
since we forgot/misunderstood before: maybe we should explain a little bit here?

Reply:
Good idea